### PR TITLE
Remove systemd (polkit + revdeps)

### PR DESCRIPTION
**Pull request by beefcurtains**
EDIT: not going to rebuild everything again

### CODE REVIEW

File: srcpkgs/lightdm/template [hunk 1]
```diff
@@ -14,15 +14,12 @@ homepage="https://launchpad.net/lightdm"
 distfiles="https://launchpad.net/lightdm/${version%.*}/$version/+download/$pkgname-$version.tar.xz"
 checksum=28b078979870782aaa1a9167945e1dab19cc6d699747af3d0c0c9a3bdb8dbba1
 
-# both install /etc/dbus-1/system.d/org.freedesktop.DisplayManager.conf
-conflicts="sddm>=0"
```
Comment (xtraeme) [inline srcpkgs/lightdm/template]: This conflict is still relevant, please re-add this back.

Comment (beefcurtains) [inline srcpkgs/lightdm/template]: Neither appear to install that file...

Comment (xtraeme) [inline srcpkgs/lightdm/template]: hmmm perhaps the file is not installed without systemd. ok

Comment (beefcurtains) [inline srcpkgs/lightdm/template]: Reverting the revert :smile:

**Comment by thypon** [comment]
Also modemmanager

**Comment by xtraeme** [comment]
I've updated polkit, please rebase.

**Pull request merged**